.github/workflows/release.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will tag, release, and publish to npm.
+name: Production Release
+
+# workflow will only run when a pull request to the release branch is closed.
+on:
+  push:
+    branches: [release]
+
+jobs:
+  release_master:
+    name: Tag and Release
+    # only run if PR is merged (not just closed)
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      # we take the message of the commit to use for tag and release body.
+      - name: get commit message
+        run: |
+          echo commitmsg=$(git log --format=%B -n 1 ${{ github.event.after }}) >> $GITHUB_ENV
+      # the tag will match the package.json version (eg. v1.0.0)
+      - name: Tag
+        id: autotagger
+        uses: butlerlogic/action-autotag@stable
+        with:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          strategy: package
+          tag_prefix: v
+
+      - name: Release
+        id: create_release
+        if: steps.autotagger.outputs.tagname != ''
+        uses: actions/create-release@v1.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: ${{ steps.autotagger.outputs.tagname }}
+          release_name: ${{ steps.autotagger.outputs.tagname }}
+          # use the body of the PR commit as the release body
+          body: "$commitmsg"
```

Comment:
Our current release process using `npm run release` creates a release on github including bullets for every PR since the last release, and linking to those commits. I quite like that feature and would like to find a way to keep something like it if we move to automated releases. This current setup will use whatever is in the body of the release commit. 

For the time being, tbh, I think it's fine to keep the manual releases unless we want to set up separate `dev` and `release` branches. I don't really see a need to do that though

Reply:
The only thing I want to make sure of is that this is something that more than one person has npm permissions to do.  But the point of the PR was to get a V1 released, and I was only going to do this if it was uncontroversial.  If we want to actually talk about it let's make a new ticket to talk about how it would work.